[2025-02-04T06:06:43.888+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-04T06:06:43.897+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: idp_fetch_mongo.fetch_from_mongo manual__2025-02-04T06:06:42.085534+00:00 [queued]>
[2025-02-04T06:06:43.902+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: idp_fetch_mongo.fetch_from_mongo manual__2025-02-04T06:06:42.085534+00:00 [queued]>
[2025-02-04T06:06:43.902+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-02-04T06:06:43.908+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): fetch_from_mongo> on 2025-02-04 06:06:42.085534+00:00
[2025-02-04T06:06:43.914+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=345) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-04T06:06:43.915+0000] {standard_task_runner.py:72} INFO - Started process 346 to run task
[2025-02-04T06:06:43.915+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'idp_fetch_mongo', 'fetch_from_mongo', 'manual__2025-02-04T06:06:42.085534+00:00', '--job-id', '145', '--raw', '--subdir', 'DAGS_FOLDER/idp.py', '--cfg-path', '/tmp/tmpz2cyn_ev']
[2025-02-04T06:06:43.916+0000] {standard_task_runner.py:105} INFO - Job 145: Subtask fetch_from_mongo
[2025-02-04T06:06:43.945+0000] {task_command.py:467} INFO - Running <TaskInstance: idp_fetch_mongo.fetch_from_mongo manual__2025-02-04T06:06:42.085534+00:00 [running]> on host 3aa1f743f988
[2025-02-04T06:06:43.995+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='idp_fetch_mongo' AIRFLOW_CTX_TASK_ID='fetch_from_mongo' AIRFLOW_CTX_EXECUTION_DATE='2025-02-04T06:06:42.085534+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-04T06:06:42.085534+00:00'
[2025-02-04T06:06:43.996+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-02-04T06:06:43.996+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-02-04T06:06:43.997+0000] {logging_mixin.py:190} INFO - Current task name:fetch_from_mongo state:running start_date:2025-02-04 06:06:43.897601+00:00
[2025-02-04T06:06:43.997+0000] {logging_mixin.py:190} INFO - Dag name:idp_fetch_mongo and current dag run status:running
[2025-02-04T06:06:43.997+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-04T06:06:44.007+0000] {base.py:84} INFO - Retrieving connection 'mongoid'
[2025-02-04T06:06:44.009+0000] {logging_mixin.py:190} INFO - Error fetching from MongoDB: MongoDB URI options are key=value pairs.
[2025-02-04T06:06:44.010+0000] {python.py:240} INFO - Done. Returned value was: []
[2025-02-04T06:06:44.024+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-04T06:06:44.024+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=idp_fetch_mongo, task_id=fetch_from_mongo, run_id=manual__2025-02-04T06:06:42.085534+00:00, execution_date=20250204T060642, start_date=20250204T060643, end_date=20250204T060644
[2025-02-04T06:06:44.029+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-02-04T06:06:44.029+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-02-04T06:06:44.029+0000] {logging_mixin.py:190} INFO - Dag name:idp_fetch_mongo queued_at:2025-02-04 06:06:42.102827+00:00
[2025-02-04T06:06:44.029+0000] {logging_mixin.py:190} INFO - Task hostname:3aa1f743f988 operator:PythonOperator
[2025-02-04T06:06:44.061+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-04T06:06:44.129+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-04T06:06:44.130+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
